### PR TITLE
get started: adjust experiments section to be 1.0 compatible

### PR DESCRIPTION
**Pull request by shcheklein**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by shcheklein** [comment]
CI fails due to the regular `ERROR:404:https://dvc.org/img/plots_roc_get_started.svg` problem with new images and links, it'll be solved when we deploy.

File: content/docs/tutorials/get-started/experiments.md [hunk 1]
```diff
@@ -1,173 +1,217 @@
...
+  plots:
+    - prc.json:
+        cache: false
 ```
 
-> Notice that we're versioning our parameters file with Git, in case we want to
-> change its contents for further experiments.
+The biggest difference is that we see two new sections `metrics` and `plots`
+that are used to mark certain files containing experiment "telemetry". Metric
+files contain scalars (e.g. `AUC`) and plots contain matrices and arrays of
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/tutorials/get-started/experiments.md:48]: plots -> plots files

File: content/docs/tutorials/get-started/experiments.md [hunk 2]
```diff
@@ -1,173 +1,217 @@
...
+numbers that are supposed to be visualized to compare or make sense of them.
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:49]: that are easier to make sense of by visualizing and comparing them.

File: content/docs/tutorials/get-started/experiments.md [hunk 3]
```diff
@@ -1,173 +1,217 @@
...
 
-Let's now redefine the featurization stage so that DVC knows that it depends on
-the specific values of `max_features` and `ngram_range`. For this we use the
-`-p` (`--params`) option of `dvc run`. `params.yaml` is the default parameters
-file name in DVC, so there's no need to specify this:
+`cache: false` means that those file are small enough and versioned directly
+with Git.
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:52]: I'd make this part of the previous p

File: content/docs/tutorials/get-started/experiments.md [hunk 4]
```diff
@@ -1,173 +1,217 @@
...
-          -p max_features,ngram_range.lo,ngram_range.hi \
-          -o data/features \
-          python src/featurization.py \
-                 data/prepared data/features params.yaml
+</details>
 
-$ git add featurize.dvc
-$ git commit -m "Update featurization stage"
+DVC expects `evaluate.py` to
+[write](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L35)
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:57]: Prob better to just link `evaluate.py` without specific line number

Comment (jorgeorpinel) [reply content/docs/tutorials/get-started/experiments.md:57]: But I don't think DVC expects anything so detail, just that the metrics file is output by this command. Maybe something like:

``[`evaluate.py`](...) should write the model's [AUC value](...) to `scores.json`, which...

File: content/docs/tutorials/get-started/experiments.md [hunk 5]
```diff
@@ -1,173 +1,217 @@
...
+```json
+{ "auc": 0.57313829 }
 ```
 
-> Please refer to `dvc params` for more information.
+And it
+[writes](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L31)
+`precision`, `recall`, `thresholds` arrays it gets with the
+[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html)
+call into the `prc.json`:
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:70]: I'd avoid this [write] link. It's kind of repeated. Also simplify to:

`precision`, `recall`, `thresholds` arrays into `prc.json` (using [`precision_recall_curve`]):

File: content/docs/tutorials/get-started/experiments.md [hunk 6]
```diff
@@ -1,173 +1,217 @@
...
+
+```json
+{
+  "prc": [
+    { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
+    ...{ "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:76]:
```suggestion
    ...
    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
```

File: content/docs/tutorials/get-started/experiments.md [hunk 7]
```diff
@@ -1,173 +1,217 @@
...
+  ]
+}
+```
 
-### Run the experiment
+> DVC doesn't force you to use any specific file names, or even format or
+> structure of a metrics or plots file - it's pretty much user and case defined,
+> please, refer to `dvc metrics` and `dvc plots` documentation for more details.
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:83]: ~~documentation~~

File: content/docs/tutorials/get-started/experiments.md [hunk 8]
```diff
@@ -1,173 +1,217 @@
...
 
-Let's [reproduce](/doc/tutorials/get-started/data-pipelines#reproduce) our
-pipeline up to the model training now:
+Let's save this experiment to compare it later using DVC comparison commands:
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:85]: ~~using DVC comparison commands~~ ? I'm not sure it will mean much to the reader until diff commands are introduced later.

Comment (jorgeorpinel) [reply content/docs/tutorials/get-started/experiments.md:85]: save this experiment to compare it with other ones later.

File: content/docs/tutorials/get-started/experiments.md [hunk 9]
```diff
@@ -1,173 +1,217 @@
...
 
-> Notice that `git commit -a` stages all the changes produced by `dvc repro`
-> before committing them with Git. Refer to the
-> [command reference](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--a)
-> for more details.
+We've managed to capture metrics that experiment has, and later we will see how
+this information can be used to compare different experiment iterations,
+visualize them (e.g. `ROC` or model loss plots). Let's see how can we capture
+another important piece of information that will be important to compare
+experiments - experiment parameters.
```
Comment (jorgeorpinel) [inline content/docs/tutorials/get-started/experiments.md:96]: metrics from an experiment. Later we...

compare different experiment iterations (e.g. `ROC` or model loss plots) and visualize them.

Comment (jorgeorpinel) [reply content/docs/tutorials/get-started/experiments.md:96]: "Let's see how can we capture" -> move this one to after the next H2?

**Comment by shcheklein** [comment]
@jorgeorpinel yes, please, create a quick PR to fix those.

File: content/docs/tutorials/get-started/experiments.md [hunk 10]
```diff
@@ -1,173 +1,217 @@
...
+        cache: false
+  plots:
+    - prc.json:
+        cache: false
 ```
 
-> Notice that we're versioning our parameters file with Git, in case we want to
-> change its contents for further experiments.
+The biggest difference is that we see two new sections `metrics` and `plots`
+that are used to mark certain files containing experiment "telemetry". Metric
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/tutorials/get-started/experiments.md:47]: Is this common DS/ML jargon?

File: content/docs/tutorials/get-started/experiments.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/tutorials/get-started/experiments.md:52]: Or a block quote. Done
